src/InputField.js

```diff
@@ -54,8 +56,8 @@ class InputField extends React.Component {
 
                 <Input
                     focus={focus}
-                    onFocus={this.onFocus}
-                    onBlur={this.onBlur}
+                    onFocus={onFocus}
```

Comment:
Let me know if this was intentional, it seemed like a typo to me. In storybook before this change, it was throwing a TypeError (`this.props.onFocus is not a function`). I assume the intention here was to allow the user of this component to specify a callback for the onFocus/onBlur events.

Reply:
Ah, so maybe the intention here was to refactor this in the same manner as the Checkbox? With class methods for onFocus and onBlur. I guess it got lost somewhere. So we'll have to implement this here just like with Checkbox?

Reply:
Good catch, @ismay. The `InputField` used to have class methods for `onFocus` and `onBlur`. Since a `InputField` should only ever be a composition of smaller components, the class methods were pushed down to `Input`: https://github.com/dhis2/ui-core/blob/next/src/Input.js#L94-L102

I forgot to change `this.onFocus` and `this.onBlur` to `props.onFocus` and `props.onBlur` for the `InputField` component. I think your change here is correct!